pkg/reconciler/cache/replication/replication_controller.go

```diff
@@ -114,12 +116,12 @@ func NewController(
 				local:  localKcpInformers.Tenancy().V1alpha1().WorkspaceTypes().Informer(),
 				global: globalKcpInformers.Tenancy().V1alpha1().WorkspaceTypes().Informer(),
 			},
-			tenancyv1alpha1.SchemeGroupVersion.WithResource("synctargets"): {
+			workloadv1alpha1.SchemeGroupVersion.WithResource("synctargets"): {
```

Comment:
Why did this not break anything in the past ... ?

Reply:
I assume nothing was using these from the cache server